frontend/src/components/utils/DropDown.tsx

```diff
@@ -21,9 +24,9 @@ const useStyles = makeStyles({
   },
 });
 
-export default function BasicMenu({ menuItems }: Props) {
+export default function BasicMenu({ menuItems, defaultValue, className, icon }: Props) {
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
-  const [selected, setSelected] = useState<string>(menuItems[0].item || '-');
+  const [selected, setSelected] = useState<string>(defaultValue ? defaultValue : 'Recent');
```

Comment:
Instead of the fixed fallback value of 'Recent', maybe the fallback value if no defaultValue is provided should be the first element in the list of options? This will account for cases where recent isn't actually an option in the dropdown.